tasks/css.js

```diff
@@ -0,0 +1,27 @@
+var bourbon = require('node-bourbon');
+var gulp = require('gulp');
+var gutil = require('gulp-util');
+var sass = require('gulp-sass');
+var sourcemaps = require('gulp-sourcemaps');
+
+var compile = function(){
+    return gulp.src('./src/sass/main.scss')
+        .pipe(sourcemaps.init())
+        .pipe(sass({
+            includePaths: bourbon.includePaths
+        }).on('error', sass.logError))
+        .pipe(sourcemaps.write(''))
+        .pipe(gulp.dest('./dist/client/static/css/'));
+};
+
+var watch = function(){
```

Comment:
Running this task will mean css will update whenever we make changes to the sass file? If so, awesome! (Y)